crates/fiber-lib/src/ckb/config.rs

```diff
@@ -76,7 +87,11 @@ impl CkbConfig {
             Ok(())
         }
     }
-
+    #[cfg(target_arch = "wasm32")]
+    pub fn read_secret_key(&self) -> Result<SecretKey> {
+        Ok(self.wasm_secret_key.expect("SecretKey not found on wasm"))
```

Comment:
is it in plain format? do we have the security issue which we introduced ENV_FIBER_SECRET_KEY_PASSWORD to resolve.

Reply:
@quake Do you have any idea about providing secret key in a more secure way on wasm? Currently it's passed through argument in plain text

Reply:
let's do it in another  PR.